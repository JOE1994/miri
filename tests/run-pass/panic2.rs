fn main() {
    let val = "Value".to_string();
    panic!("Miri panic with value: {}", val);
}
