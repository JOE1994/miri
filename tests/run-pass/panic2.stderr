thread 'main' panicked at 'Miri panic with value: Value', $DIR/panic2.rs:3:5
